lib/rubocop/cop/sorbet/single_line_rbi_class_module_definitions.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # This cop ensures empty class/module definitions in RBI files are
+      # done on a single line rather than being split across multiple lines.
+      #
+      # @example
+      #
+      #   # bad
+      #   module SomeModule
+      #   end
+      #
+      #   # good
+      #   module SomeModule; end
+      class SingleLineRbiClassModuleDefinitions < RuboCop::Cop::Cop
+        MSG = 'Empty class/module definitions in RBI files should be on a single line.'
+
+        def on_module(node)
+          process_node(node)
+        end
+
+        def on_class(node)
+          process_node(node)
+        end
+
+        def autocorrect(node)
+          -> (corrector) { corrector.replace(node, convert_newlines(node.source)) }
+        end
+
+        protected
+
+        def convert_newlines(source)
+          source.sub("\n", "; ").tr("\n", "")
```

Comment:
What if the bock contains a space like this:

```rb
class Foo
  # <- empty indented line
end
```

Are we letting this being caught by another cop?

Reply:
Suggestion for handling the multiple empty lines:
```ruby
source.gsub(/[\r\n]+/, "; ")
```

Reply:
Ah good eye! I'll see if I can make this a bit more clever.